libraries/chef_ingredient_resource.rb

```diff
@@ -17,17 +17,23 @@
 class Chef
   class Resource
     class ChefIngredient < Chef::Resource::LWRPBase
+      begin
+        require 'mixlib/install'
+      rescue LoadError
+        ensure_mixlib_install_gem_installed!
+      end
```

Comment:
Is there some reason we placed the `require` down here in the class vs at the top of the file as idiomatic Ruby would say we should?